tests/test_layers.py

```diff
@@ -0,0 +1,19 @@
+import tensorflow as tf
+from tensorflow.python.keras.utils.tf_utils import ListWrapper
+from ext.lab2im.edit_tensors import gaussian_kernel
+
+
+def test_gaussian_blur2():
```

Comment:
Just noticed, that test also fails for TF 2.2 in my "synth_seg_py38" environment.